rules/stylelistic.js

```diff
@@ -36,13 +36,8 @@ const rules = {
   'one-var-declaration-per-line': ['warn', 'always'],
   'padded-blocks': ['warn', 'never'],
   'quotes': ['warn', 'single'],
-  'require-jsdoc': ['warn', {
-    'require': {
-      'FunctionDeclaration': true,
-      'MethodDefinition': true,
-      'ClassDeclaration': false
-    }
-  }],
+  'valid-js': 'off',
+  'require-jsdoc': 'off',
```

Comment:
why are we removing require-js doc... we still require one right?

Can't we just configure it to check that there is a doc there?